src/applications/coronavirus-vaccination-expansion/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "appName": "covid-vaccine-expansion",
```

Comment:
The same case here, if we update the `appName` in the `registry.json` file we'd probably want to do that here as well.